sstables/sstables.hh

```diff
@@ -134,6 +135,7 @@ public:
     using version_types = sstable_version_types;
     using format_types = sstable_format_types;
     using tracker_link_type = bi::list_member_hook<bi::link_mode<bi::auto_unlink>>;
```

Comment:
I think the sstable manager can take on the role of the tracking -- scylla-gdb.py can use the manager to find all sstables on a shard, no need for a separate debug list. No need to do in this series.

Reply:
Note we have two managers per shard now, and may have more later.